docs/pages/upcoming-releases.mdx

```diff
@@ -0,0 +1,343 @@
+      <td>Use `variant` instead</td>
+    </tr>
+    <tr>
+      <td>`Card`</td>
+      <td>`variant`</td>
+      <td>`migrateCardVariantToColor`</td>
+      <td>Use `color` instead</td>
+    </tr>
+    <tr>
+      <td>`ConfirmationModal`</td>
+      <td>`mood`</td>
+      <td>`migrateConfirmationModalMoodsToVariant`</td>
+      <td>Use `variant` instead</td>
+    </tr>
+    <tr>
+      <td>`EmptyState`</td>
+      <td>`illustrationType`</td>
+      <td>`migrateEmptyStateIllustrationTypeToVariant`</td>
+      <td>Use `variant` instead</td>
+    </tr>
+    <tr>
+      <td>`InputEditModal`</td>
+      <td>`mood`</td>
+      <td>`removeInputEditModalMood`</td>
+      <td>
+        We are no longer supporting different moods for InputModal, instead there will only be a
+        single default variant set by the component.
+      </td>
+    </tr>
+    <tr>
+      <td>`Popover`</td>
+      <td>`variant`</td>
+      <td>`removePopoverVariant`</td>
+      <td>
+        We are no longer supporting different variants for Popover, instead there will only be a
+        single default variant.
+      </td>
+    </tr>
+    <tr>
+      <td>`Popover`</td>
+      <td>`customIcon`</td>
+      <td>-</td>
+      <td>This was for customising the icon provided with variants.</td>
+    </tr>
+    <tr>
+      <td>`ProgressBar`</td>
+      <td>`mood`</td>
+      <td>`migrateProgressBarMoodToColor`</td>
+      <td>Use `color` prop instead</td>
+    </tr>
+    <tr>
+      <td>`InformationTile`</td>
+      <td>`mood`</td>
+      <td>`migrateInformationTileMoodToVariant`</td>
+      <td>Use `variant` instead</td>
+    </tr>
+    <tr>
+      <td>`MultiActionTile`</td>
+      <td>`mood`</td>
+      <td>`migrateMultiActionTileMoodToVariant`</td>
+      <td>Use `variant` instead</td>
+    </tr>
+    <tr>
+      <td>`Well`</td>
+      <td>`variant`</td>
+      <td>`migrateWellVariantToColor`</td>
+      <td>
+        This will not fallback to default variant. default can be used but must be explicitly passed
+        to the Well component. It is recommended to use color prop and gray value if you need a gray
+        background.
+      </td>
+    </tr>
+    <tr>
+      <td colSpan={4}>automationId</td>
+    </tr>
+    <tr>
+      <td>`LikertScaleLegacy`</td>
+      <td>`automationId`</td>
+      <td>-</td>
+      <td>Use data-testid instead</td>
+    </tr>
+    <tr>
+      <td>`ConfirmationModal`</td>
+      <td>☝️</td>
+      <td>-</td>
+      <td>☝️</td>
+    </tr>
+    <tr>
+      <td>`ConfirmationModal`</td>
+      <td>☝️</td>
+      <td>-</td>
+      <td>☝️</td>
+    </tr>
+    <tr>
+      <td>`ContextModal`</td>
+      <td>☝️</td>
+      <td>-</td>
+      <td>☝️</td>
+    </tr>
+    <tr>
+      <td>`InputEditModal`</td>
+      <td>☝️</td>
+      <td>-</td>
+      <td>☝️</td>
+    </tr>
+    <tr>
+      <td>`ModalFooter`</td>
+      <td>☝️</td>
+      <td>-</td>
+      <td>☝️</td>
+    </tr>
+    <tr>
+      <td colSpan={4}>Others</td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
Just a note, I may have some more props that will be deprecated with all this button internal work. I reckon it'd probs be best to hold until the following release give the shorter lead time but happy to hear thought :)

Reply:
Also on that note, do you think it'll be ok re: the removal of v1 Button / IconButton to only remove it as a exported component? We may need to hang onto this until all internal uses are gone in its entirety.

Reply:
Oops just realised I forgot to add the 'others' here

Reply:
>  I may have some more props that will be deprecated with all this button internal work

Yeah I reckon any Button stuff (e.g. `reversed`) should wait for the next cycle so that this one can just focus on getting people over to the new button.

> Also on that note, do you think it'll be ok re: the removal of v1 Button / IconButton to only remove it as a exported component? 

Ya that seems like it would be fine as far as I can tell